crates/handlers/src/upstream_oauth2/template.rs

```diff
@@ -25,13 +25,19 @@ fn split(value: &str, separator: Option<&str>) -> Vec<String> {
 }
 
 fn b64decode(value: &str) -> Result<Value, Error> {
-    let bytes = Base64::decode_vec(value).map_err(|e| {
-        Error::new(
-            ErrorKind::InvalidOperation,
-            "Failed to decode base64 string",
-        )
-        .with_source(e)
-    })?;
+    // We're not too concerned about the performance of this filter, so we'll just
```

Comment:
This feels a bit lax TBH, I'm not sure about the intended usage but this is the kind of thing that makes you accidentally accept something you didn't mean to and then make it hard to break that later.

If we're really sure this is fine then fine, but otherwise I'd prefer to have different filters for the different variants

Reply:
This *can* be used when transforming attributes from an upstream IDP. We use this for decoding the Dex subject in ESS, and as we're not documenting this, I think it'll be fine if we want to break it later